### ISSUE TITLE
Eye tracking not working after upgrading Varjo Base to 4.3.0.14

### ISSUE THREAD

**Issue by nerdaxic** [Chickenbreadlp/VRCFTVarjoModule#2]
After upgrading Varjo Base from 3.10.3.25 to 4.3.0.14, eye tracking no longer works in VRCFT. The VRCFT UI displays the Aero-icon with a question mark and the text "Varjo Eye Tracking".

**Steps to Reproduce:**
1. Upgrade Varjo Base from version 3.10.3.25 to 4.3.0.14.
2. Launch VRCFT.
3. Observe the error in the VRCFT UI regarding eye tracking. Eye tracking does not work.

**Expected Behavior:**
Eye tracking should function properly in VRCFT after upgrading Varjo Base.

**Actual Behavior:**
The eye tracking feature does not work, and the VRCFT UI displays an error message.
Error in Output: `Unknown HMD Name: X002`

**Environment:**
Headset: XR-3
OS: Windows 10 N 64-bit
Varjo Base version: 4.3.0.14 (Issue occurs), 3.10.3.25 (Works fine)
VRCFT Version: 5.2.3.0

**Additional Context:**
- Possible changes to Varjo's Eye tracking SDK may have caused this issue.
- Eye tracking is confirmed to be enabled in Varjo Base.
- Downgrading Varjo Base to 3.10.3.25 resolves the issue.
- Varjo has since 3.10.3.25 published XR-4 headset, possibly changing under-the-hood naming scheme of the headsets.

Output when starting VRCFT:
```
[VRCFaceTracking.Services.ActivationService] Information: VRCFT Version 5.2.3.0 initializing...
[VRCFaceTracking.Services.ActivationService] Information: Initializing OSC...
[VRCFaceTracking.Core.OSC.OscQueryService] Debug: OSC Service Initializing
[VRCFaceTracking.Core.OSC.OscQueryService] Debug: OSC Service Initialized with result (False, False)
[VRCFaceTracking.Services.ActivationService] Information: Initializing main service...
[VRCFaceTracking.Core.Params.Data.UnifiedTrackingMutator] Debug: Reading configuration...
[VRCFaceTracking.Core.MainStandalone] Debug: Starting OSC update loop...
[VRCFaceTracking.Services.ActivationService] Debug: Checking for deletion requests for installed modules...
[VRCFaceTracking.Services.ActivationService] Information: Checking for updates for installed modules...
[VRCFaceTracking.Core.Params.Data.UnifiedTrackingMutator] Debug: Configuration loaded.
[VRCFaceTracking.Services.ActivationService] Information: Initializing modules...
[VRCFaceTracking.Core.Library.UnifiedLibManager] Information: Starting initialization tracking
[VRCFaceTracking.Core.Library.UnifiedLibManager] Information: Tearing down all modules...
[Microsoft.Extensions.Hosting.Internal.Host] Debug: Hosting starting
[VRCFaceTracking.Core.Library.UnifiedLibManager] Debug: SRanipalExtTrackingInterface properly implements ExtTrackingModule.
[VRCFaceTracking.Core.Library.UnifiedLibManager] Debug: VarjoTrackingModule properly implements ExtTrackingModule.
[VRCFaceTracking.Core.Library.UnifiedLibManager] Debug: Initializing requested runtimes...
[VRCFaceTracking.Core.Library.UnifiedLibManager] Information: Initializing runtimes...
[VRCFaceTracking.Core.Library.UnifiedLibManager] Information: Initializing SRanipalExtTrackingModule, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null
[VRCFaceTracking.Core.Library.UnifiedLibManager] Information: Loading External Module SRanipalExtTrackingModule, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null
[Microsoft.Hosting.Lifetime] Information: Application started. Press Ctrl+C to shut down.
[Microsoft.Hosting.Lifetime] Information: Hosting environment: Production
[Microsoft.Hosting.Lifetime] Information: Content root path: C:\Program Files\WindowsApps\96ba052f-0948-44d8-86c4-a0212e4ae047_5.2.3.0_x64__4s4k90pjvq32p\
[Microsoft.Extensions.Hosting.Internal.Host] Debug: Hosting started
[SRanipalExtTrackingInterface] Information: SRanipalExtTrackingModule: SRanipal version: 1.3.1.1
[SRanipalExtTrackingInterface] Information: Initializing Eye...
[SRanipalExtTrackingInterface] Information: Eye failed to initialize: NOT_SUPPORT_EYE_TRACKING
[SRanipalExtTrackingInterface] Information: Initializing Lip...
[SRanipalExtTrackingInterface] Information: Lip successfully started!
[VRCFaceTracking.Core.Library.UnifiedLibManager] Debug: Starting thread for SRanipalExtTrackingInterface
[VRCFaceTracking.Core.Library.UnifiedLibManager] Information: Initializing VRCFTVarjoModule, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null
[VRCFaceTracking.Core.Library.UnifiedLibManager] Information: Loading External Module VRCFTVarjoModule, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null
[VarjoTrackingModule] Information: Loaded library C:\Users\nerdaxic\AppData\Roaming\VRCFaceTracking\CustomLibs\d1bf4744-80c4-4cba-849e-22940511901e\TrackingLibs\VarjoLib.dll
[VarjoTrackingModule] Information: Unknown HMD Name: X002
[VRCFaceTracking.Core.Library.UnifiedLibManager] Debug: Starting thread for VarjoTrackingModule
[VRCFaceTracking.Core.Library.UnifiedLibManager] Information: Tracking initialized via SRanipalExtTrackingInterface.SRanipalExtTrackingInterface
[VRCFaceTracking.Core.Library.UnifiedLibManager] Information: Tracking initialized via VRCFTVarjoModule.VarjoTrackingModule
[VRCFaceTracking.Core.Services.ModuleDataService] Debug: Failed to get user rating for module 74f18332-1bcf-4b55-832d-9c91069afa31
```

**Comment by Chickenbreadlp** [comment]
I think this is two seperate issues colliding at the same time here:
- The Questionmark Aero headset just means the module couldn't identify the headset model. Starting with VB4.3 the API stopped delivering human readable device names. It doesn't reflect the actual tracking status
- The not working eye tracking I'm not sure tbh. VB4.3 is a lot more strict when it comes to reflections from glasses or lens inserts. Is ET calibration working fine?

**Comment by nerdaxic** [comment]
I have confirmed [beta version](https://github.com/Chickenbreadlp/VRCFTVarjoModule/releases/tag/4.1.0-beta1) to work with 4.3.0.14 and XR-3.

Seems to be an issue like you said:

- Icon just being an icon, broken with Varjo Base V4.3 update.
- Much more finicky eye calibration needs to be passed

I have prescription lens inserts from VROptician installed, which can complicate stuff. 
I also add about 5mm manually to my measured IPD because it basically removes the red-blue shift in Varjo headsets. (it thinking about it, read about Varjo's `OpticalAdjustment.exe`)

**Heres what I did to get it working:**
1) Upgraded Varjo Base back to 4.3.0.14
2) Installed [VRCFTVarjoModule BETA ](https://github.com/Chickenbreadlp/VRCFTVarjoModule/releases/tag/4.1.0-beta1) into VRCFT 5.2.3.0
3) Successful eye calibration in Varjo Base
4) Avatar and OSC reset in VRChat
5) I can make stupid faces again

Todo: Cook machine readible ID:s into correct icons.
X002 seems to be XR-3
